releasenotes/v1.10.0-beta.0.md

```diff
@@ -0,0 +1,170 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v1.9.0
+
+## :warning: Breaking Changes
+
+- Bump CAPI to v1.10.0-rc.0 (#2487)
```

Comment:
I would suggest to keep `Bump CAPI to v1.10.0-beta.0 and Controller Runtime to 0.20.3 (#2432) `
here, #2487 is not breaking change. Let's not move #2432 to Superseded, it has other changes too.
We can move #2487 to others.

Reply:
Agree with @adilGhaffarDev otherwise lgtm

Reply:
Thanks, done!